routes/index.js

```diff
@@ -27,11 +27,16 @@ async function getGraphQLTableSchema(resource_id) {
   }
 `
   // return request(`${process.env.HASURA_URL}/v1/graphql`, queryForSchema)
+  const graphQLClient = new GraphQLClient(
+    `${process.env.HASURA_URL}/v1/graphql`,
+    {
+      headers: {
+        'x-hasura-admin-secret': process.env.HASURA_GRAPHQL_ADMIN_SECRET,
```

Comment:
@luccasmmg doesn't it mean that users now can mutate data if the secret is set?

Reply:
That's why I made sure to only add the header to the datastore_search route, because on that one the query is already written, the other ones are still done using the anonymous user